modules/reference/pages/cluster-properties.adoc

```diff
@@ -1,1796 +1,5372 @@
+
+*Type*: boolean.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Type:* boolean
+
+*Default:* `true`
+
+---
+
+=== enable_leader_balancer
+
+Enable automatic leadership rebalancing. Mode is set by <<leader_balancer_mode,`leader_balancer_mode`>>.
+
+*Type*: boolean.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Type:* boolean
+
+*Default:* `true`
+
+---
+
+=== enable_metrics_reporter
+
+Enable the cluster metrics reporter. If `true`, the metrics reporter collects and exports to Redpanda Data a set of customer usage metrics at the interval set by xref:./tunable-properties.adoc#metrics_reporter_report_interval[metrics_reporter_report_interval].
+
+[NOTE]
+====
+The cluster metrics of the metrics reporter are different from xref:manage:monitoring.adoc[monitoring metrics].
+
+* The metrics reporter exports customer usage metrics for consumption by Redpanda Data.
+* Monitoring metrics are exported for consumption by Redpanda users to monitor their system's health.
+====
+
+*Type*: boolean.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Type:* boolean
+
+*Default:* `true`
+
+---
+
+=== enable_mpx_extensions
+
+Enable Redpanda extensions for MPX.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* boolean
+
+*Default:* `false`
+
+---
+
+=== enable_pid_file
+
+Enable pid file. You probably don't want to change this.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* boolean
+
+*Default:* `true`
+
+---
+
+=== enable_rack_awareness
+
+Enable rack-aware replica assignment.
+
+*Type*: boolean.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Type:* boolean
+
+*Default:* `false`
+
+---
+
+=== enable_sasl
+
+Enable SASL authentication for Kafka connections.
+
+*Type*: boolean.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Type:* boolean
+
+*Default:* `false`
+
+---
+
+=== enable_schema_id_validation
+
+Mode to enable server-side schema ID validation.
+
+*Valid values*:
+
+* `none`: schema validation is disabled (no schema ID checks are done). Associated topic properties cannot be modified.
+* `redpanda`: schema validation is enabled. Only Redpanda topic properties are accepted.
+* `compat`: schema validation is enabled. Both Redpanda and compatible topic properties are accepted.
+
+
+
+*Related topics*:
+
+* xref:manage:schema-id-validation.adoc[Server-Side Schema ID Validation].
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Default:* `pandaproxy::schema_registry::schema_id_validation_mode::none`
+
+---
+
+=== enable_transactions
+
+Enable transactions (atomic writes).
+
+*Type*: boolean.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Type:* boolean
+
+*Default:* `true`
+
+---
+
+=== enable_usage
+
+Enables the usage tracking mechanism, storing windowed history of kafka/cloud_storage metrics over time.
```

Comment:
ENG: I don't understand this. Does this only enable certain metrics, like the cloud_storage_xxx ones?